modules/adding-tls-termination.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination_{context}"]
+= Adding the tls termination on the AWS Load Balancer
+
+You can route the traffic for the domain `example.com` to pods of a `service` `example-service`, and add TLS termination on the AWS Load Balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
++
+[source,yaml]
+----
+kind: AWSLoadBalancerController
+group: networking.olm.openshift.io/v1alpha1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto
+  ingressClass: tls-termination <3>
+----
+
+<1> Defines the resource group of the `AWSLoadBalancerControlller` resource.
+<2> Defines the `aws-load-balancer-controller` instance.
+<3> Defines the name of a default `IngressClass`. The `aws-load-balancer-controller` reconciles a default `IngressClass`.
```

Comment:
Default? The default ingress class is denoted by a special annotation on the ingress class (`ingressclass.kubernetes.io/is-default-class`), it's kinda out of scope of this operator.
I would just say that this defines the ingress class to be reconciled by the controller.

Reply:
Ack'ed